cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ApplicationCredentialDTO.java

```diff
@@ -0,0 +1,55 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import javax.validation.constraints.NotNull;
+
+public class ApplicationCredentialDTO
+{
+    @NotNull
+    private String key;
+
+    private String value;
+
+    @NotNull
+    private Integer weight;
+
+    public ApplicationCredentialDTO( @NotNull String key, @NotNull String value, Integer weight )
+    {
+        this.key = key;
+        this.value = value;
+        this.weight = weight;
+    }
+
+    public ApplicationCredentialDTO()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "key" is marked "javax.validation.constraints.NotNull" but is not initialized in this constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS2637)